example/go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/priteshbandi/notation-plugin-base/example
```

Comment:
Need to remove the go.mod and go.sum, because of the main.go and plugin.go already in notation-plugin-framework-go project, so they can invoke the code directly.

Reply:
And go does't support nested projects. Its hard to maintain.

Reply:
Its more of an example sgowing how users can create plugin and the easiest way I could find was creating sub project.
Adding it to `notation-plugin-framework-go` might confuse user.